dashboard/config/scripts/levels/CSD U3 platform player2_2018.level

```diff
@@ -46,7 +46,7 @@
     "name_suffix": "_2018",
     "contained_level_names": null,
     "preload_asset_list": null,
-    "long_instructions": "# User Controls\r\n\r\nNext, you should add user controls so that you can move your player around.  Your player needs to move to the left when the left arrow key is pressed, to the right when the right arrow key is pressed, and jump when the space bar is pressed.\r\n\r\n# Do This\r\n\r\n* Create a new `controlPlayer` function in the \"functions\" area of your code.\r\n* Inside the `controlPlayer` function, use the `if`, `keyDown`, and `sprite.x` blocks to make your player move to the left and right according to the arrow keys.  Look at <a href=\"/s/csd3-2018/stage/12/puzzle/7\" target=\"_blank\"> Lesson 12 Puzzle 7 </a> for examples.\r\n* Inside the `controlPlayer` function, use the `if`, `keyDown`, and `velocityY` blocks to make your player jump when the up arrow is pressed.  Look at <a href=\"/s/csd3-2018/stage/15/puzzle/11\" target=\"_blank\"> Lesson 15 Puzzle 11 </a> for an example.\r\n* Call the `controlPlayer` function inside the draw loop.\r\n* Run the game and test your code."
+    "long_instructions": "# User Controls\r\n\r\nNext, you should add user controls so that you can move your player around.  Your player needs to move to the left when the left arrow key is pressed, to the right when the right arrow key is pressed, and jump when the space bar is pressed.\r\n\r\n# Do This\r\n\r\n* Create a new `controlPlayer` function in the \"functions\" area of your code.\r\n* Inside the `controlPlayer` function, use the `if`, `keyDown`, and `sprite.x` blocks to make your player move to the left and right according to the arrow keys.  Look at <a href=\"/s/csd3-2018/lessons/12/levels/7\" target=\"_blank\"> Lesson 12 Puzzle 7 </a> for examples.\r\n* Inside the `controlPlayer` function, use the `if`, `keyDown`, and `velocityY` blocks to make your player jump when the up arrow is pressed.  Look at <a href=\"/s/csd3-2018/lessons/15/levels/11\" target=\"_blank\"> Lesson 15 Puzzle 11 </a> for an example.\r\n* Call the `controlPlayer` function inside the draw loop.\r\n* Run the game and test your code."
```

Comment:
not blocking for this PR, but these instances of `target=\"_blank\"` should have `rel="noopener noreferrer"` on them: https://codedotorg.atlassian.net/browse/PLAT-1028